dags.yaml

```diff
@@ -1097,3 +1097,24 @@ bqetl_fenix_external:
   tags:
     - impact/tier_1
     - repo/bigquery-etl
+bqetl_fivetran_apple_ads:
+  default_args:
+    depends_on_past: false
+    email:
+      - telemetry-alerts@mozilla.com
+      - frank@mozilla.com
+      - kik@mozilla.com
+    email_on_failure: true
+    email_on_retry: true
+    end_date: null
+    owner: kik@mozilla.com
+    retries: 2
+    retry_delay: 30m
+    start_date: '2023-01-01'
+    description: |
+      Copies over apple_ads data coming from Fivetran
+      into our data BQ project. Fivetran syncs this data
+      every hour. We copy the data every 3 hours to our project.
+  schedule_interval: 0 */3 * * *
```

Comment:
Seems a little excessive, but if the data is tiny it doesn't matter

Reply:
Especially since the tables are being copied in their entirety each time.  Is the data tiny?

Reply:
My inclination would be to start with a daily schedule and only sync more frequently if a concrete use case for that comes up.